google/cloud/forseti/common/gcp_api/cloud_resource_manager.py

```diff
@@ -346,8 +361,12 @@ def get_project_org_policies(self, project_id):
         try:
             paged_results = self.repository.projects.list_org_policies(
                 resource_id)
-            return api_helpers.flatten_list_results(paged_results,
-                                                    'policies')
+            flattened_results = api_helpers\
```

Comment:
Please don't use \ for line continuation.

Reply:
replaced all \s with brackets